modules/ossm-threescale-applying-external-service-entry-objects.adoc

```diff
@@ -31,14 +32,60 @@ spec:
   location: MESH_EXTERNAL
   resolution: DNS
 ----
+.. Add the `DestinationRule` CR to a file called _destination-rule-threescale-saas-backend.yml:_
 +
-.Custom resource for 3scale Hosted system
+.DestinationRule CR
+[source,terminal]
+----
+apiVersion: networking.istio.io/v1beta1
+kind: DestinationRule
+metadata:
+  name: destination-rule-threescale-saas-backend
+spec:
+  host: su1.3scale.net
+  trafficPolicy:
+    tls:
+      mode: SIMPLE
+      sni: su1.3scale.net
+----
+
+. Apply and save the external `ServiceEntry` CR for the 3scale Hosted backend to your cluster, by running the following command:
++
+[source,terminal]
+----
+$ oc apply -f <service-entry-threescale-saas-backend.yml>
+----
+
+.. Alternatively, apply the external `ServiceEntry` CR for the 3scale Hosted backend to your cluster without first saving the file, by running the following command:
```

Comment:
This command as far as I can tell would require a previously saved `service-entry-threescale-saas-backend.yml` file and it just applies the configuration in much the same way as the preceding command. You could remove this alternative step entirely to minimise the procedure.